uaclient/cli.py

```diff
@@ -1553,7 +1553,14 @@ def action_status(args, *, cfg):
     show_beta = args.all if args else False
     token = args.simulate_with_token if args else None
     active_value = ua_status.UserFacingConfigStatus.ACTIVE.value
-
+    try:
+        if contract.is_contract_changed(cfg):
```

Comment:
Let's also only check this if the user is attached, otherwise this call will not do anything for the user